posthog/models/event/sql.py

```diff
@@ -86,7 +98,7 @@
     table_name=EVENTS_DATA_TABLE(),
     cluster=settings.CLICKHOUSE_CLUSTER,
     engine=EVENTS_DATA_TABLE_ENGINE(),
-    extra_fields=KAFKA_COLUMNS,
+    extra_fields=KAFKA_COLUMNS + ", inserted_at DateTime64(6, 'UTC') DEFAULT NOW()",
```

Comment:
Can we add this to `EVENTS_TABLE_BASE_SQL`. What's the reason for not adding there?

Reply:
That query is used as the base of the kafka table too and Kafka tables cannot have DEFAULT expressions

Reply:
So, if we add it, migration 0025 fails when trying to create the Kafka table.